### ISSUE TITLE
Compute validation loss while training?

### ISSUE THREAD

**Issue by doantientai** [affinelayer/pix2pix-tensorflow#122]
I would like to separate my dataset into 3 sets: train, val and test, so that I can use the "val" dir for validating during the training phase and "test" dir for the final test. 

From the Readme.md I think here "val" is used as a test set instead of a validation set. If I would like to make val and test separated as I said above, what can I do?

Thank you very much!

**Comment by heidotrofimov** [comment]
I tried to this, take a look:
https://github.com/affinelayer/pix2pix-tensorflow/issues/200
